packages/client/src/components/Navigation/Navbar.component.js

```diff
@@ -3,13 +3,14 @@ import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import { MobileNavigation } from '../MobileNavigation/MobileNavigation.component';
 import './Navbar.css';
+import { UserAuth } from '../../firebase/AuthContext';
 
-export const Navbar = ({ isLogedIn, userName, ...props }) => {
+export const Navbar = ({ ...props }) => {
   const [isMobileNavOpen, setIsMobileNavOpen] = useState(false);
-
+  const { googleSignIn, logOut, user } = UserAuth();
   let accountStatus = null;
 
-  if (isLogedIn) {
+  if (user) {
```

Comment:
nit: what do you think about creating components for logged in and logged out and put the code here into these components?

Reply:
Could be a good idea. We have only a variable inside of each statement. Is it okay?

Reply:
could you elaborate more about your response? what problems could having one variable cause?

Reply:
I am a bit unsure on how to implement it. Do I get right that I simply make  a React component that returns a JSX element for each state? We also have context variables that are used in the these components. Should I just pass them as props?

Reply:
yes, this is what I mean. have two components one is LoggedInUser other is GuestUser and hide the JSX and logic used for both in these two components. also, nit means nitty gritty which means that this is nice to have but it's perfectly okay to leave it as is :)

Reply:
@MahmoudYounes Done ✅ 
May you please check and approve our PR?